Canvas/Canvas/Calendar/UI/View Controller/CalendarMonthViewController.swift

```diff
@@ -260,7 +260,7 @@ open class CalendarMonthViewController: UIViewController, CalendarViewDelegate,
     fileprivate var eventsDisposable: Disposable?
     
     @objc func updateCalendarEvents() {
-        let startDate = Date() + -365.daysComponents
+        let startDate = Date() + -965.daysComponents
```

Comment:
🤞

Reply:
oops, that was me trying to reproduce the bug on Friday.